### ISSUE TITLE
Support for Christmas nicknames needed

### ISSUE THREAD

**Issue by oherik** [cthit/hubbIT-old#154]
As is customary, now is the time that people change their nicknames to their Christmas version. Unfortunately it's difficult to do this on Hubbit. Would it be possible to implement a nickname feature, where people can select an optional nickname?

**Comment by jesperlindstrom** [comment]
I believe it's already possible, since the nicknames are retrieved from chalmers.it/LDAP, but I'm guessing that we have a somewhat permanent cache in hubbIT itself. Maybe we could simply set a reasonable expiration of the nickname cache to solve this issue.

**Comment by lindskogen** [comment]
Yes, 24 hours: https://github.com/cthit/hubbIT/blob/develop/app/models/user.rb#L44
